### PR TITLE
[Task #68320]: Xác thực email

### PR DESCRIPTION
**Pull request by BuiMinhDung711**
## Related Tickets:
- [#68320](https://edu-redmine.sun-asterisk.vn/issues/68320)

## WHAT this PR does?

- Email verification feature implement
- Fix some version errors in components

## Notes Impacted Areas:
- ```app/Http/Controllers/ProductController.php```
- ```app/Http/Kernel.php```
- ```app/Http/Middleware```
- ```app/Models/User.php```
- ```app/View/Components```
- ```routes/web.php```

## Attachments:
- 
image
- 
image

### CODE REVIEW

**Comment by hoangnbvn** [comment]
ok

**Comment by BuiMinhDung711** [comment]
ready

File: app/Http/Controllers/ProductController.php [hunk 1]
```diff
@@ -24,8 +24,7 @@ public function index()
             ->select(['products.name', 'price', 'products.id', 'brand','media_link'])
             ->where('brand', '=', 'Comme des Garcons')
             ->get();
-            
-        return view('welcome', [
+        return view('home', [
```
Comment (ghost) [inline app/Http/Controllers/ProductController.php:28]: add empty line before return

File: app/Http/Middleware/CheckAccessForGuestAndVerifiedUser.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure(\Illuminate\Http\Request): (\Illuminate\Http\Response|\Illuminate\Http\RedirectResponse)  $next
+     * @return \Illuminate\Http\Response|\Illuminate\Http\RedirectResponse
+     */
+    public function handle(Request $request, Closure $next, $redirectToRoute = null)
+    {
+        if (Auth::check()) {
+            if ($request->user() instanceof MustVerifyEmail && !$request->user()->hasVerifiedEmail()) {
+                return $request->expectsJson()
+                    ? abort(403, 'Your email address is not verified.')
```
Comment (ghost) [inline app/Http/Middleware/CheckAccessForGuestAndVerifiedUser.php]: localization

**Comment by BuiMinhDung711** [comment]
ready

**Pull request merged**